sql/load_demo_data.sql

```diff
@@ -2979,6 +3097,10 @@ CREATE TABLE `oh_medicaldsrstockmov` (
   KEY `FK_MEDICALDSRSTOCKMOV_MEDICALDSR` (`MMV_MDSR_ID`),
   KEY `FK_MEDICALDSRSTOCKMOV_MEDICALDSRSTOCKMOVTYPE` (`MMV_MMVT_ID_A`),
   KEY `FK_MEDICALDSRSTOCKMOV_WARD_idx` (`MMV_WRD_ID_A`),
+  KEY `FK_MEDICALDSRSTOCKMOV_CREATED_BY_IDX` (`MMV_CREATED_BY`),
```

Comment:
In this table the column `MMV_WRD_ID_A` has the default value in the demo data loading as `NULL` where the multiple steps sets it to '' (empty string).

Reply:
For this the correct default value should be NULL, unfortunately an oversight in step_80 changing the field to CHAR(3) set it to ''.
I think address the change in OP-1107 with a new step_XX. I will report under the issue.
Thanks!